src/SourceCode.Clay/NumberKinds.cs

```diff
@@ -0,0 +1,33 @@
+#region License
+
+// Copyright (c) K2 Workflow (SourceCode Technology Holdings Inc.). All rights reserved.
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+
+#endregion
+
+using System;
+
+namespace SourceCode.Clay
+{
+#pragma warning disable S2342 // Enumeration types should comply with a naming convention
+#pragma warning disable S2346 // Flags enumerations zero-value members should be named "None"
```

Comment:
Don't ignore, add the None.

Reply:
There's an existing issue #92 to look at this. I will just remove pragma so we can fix later